All_home_works/hw1/task4.py

```diff
@@ -0,0 +1,30 @@
+"""
+Given a list of integers numbers "nums".
+You need to find a sub-array with length less equal to "k", with maximal sum.
```

Comment:
What does `less equal to "k"` mean?

Reply:
I got it. I can suggest solving this, but with complexity (n*k), is it ok?